<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-8C0LO3K" failures="1" tests="1" name="Subscription Test" time="7.231" errors="0" timestamp="2024-10-10T12:26:23 IST">
  <testcase classname="com.demoshop.tests.SubscriptionTest" name="verifyUserIsAbleToSubscribeWithValidEmail" time="0.819">
    <failure type="java.lang.AssertionError" message="expected [Thank you for signing up! A verification email has been sent. We appreciate your interest.] but found [hvhjvhj]">
      <![CDATA[java.lang.AssertionError: expected [Thank you for signing up! A verification email has been sent. We appreciate your interest.] but found [hvhjvhj]
at com.demoshop.tests.SubscriptionTest.verifyUserIsAbleToSubscribeWithValidEmail(SubscriptionTest.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- verifyUserIsAbleToSubscribeWithValidEmail -->
</testsuite> <!-- Subscription Test -->
